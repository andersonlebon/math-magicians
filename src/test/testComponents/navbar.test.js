import React from 'react';
import { HashRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Navbar from '../../components/navbar';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <HashRouter>
        <Navbar />
      </HashRouter>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
