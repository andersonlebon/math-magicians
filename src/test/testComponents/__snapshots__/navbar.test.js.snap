// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<header
  className="w-100"
>
  <nav
    className="navbar navbar-expand-lg navbar-dark w-100 bg-dark"
  >
    <h1
      className="navbar-brand font"
    >
      Math magicians
    </h1>
    <button
      aria-controls="navbarNavAltMarkup"
      aria-expanded="false"
      aria-label="Toggle navigation"
      className="navbar-toggler"
      data-target="#navbarNavAltMarkup"
      data-toggle="collapse"
      type="button"
    >
      <span
        className="navbar-toggler-icon"
      />
    </button>
    <div
      className="collapse navbar-collapse"
      id="navbarNavAltMarkup"
    >
      <div
        className="navbar-nav ml-auto"
      >
        <a
          className="nav-item nav-link active font"
          href="#/"
          onClick={[Function]}
        >
          Home
          <span
            className="sr-only"
          >
            (current)
          </span>
        </a>
        <a
          className="nav-item nav-link font"
          href="#/calculator"
          onClick={[Function]}
        >
          Calculator
        </a>
        <a
          className="nav-item nav-link font"
          href="#/quotes"
          onClick={[Function]}
        >
          Quotes
        </a>
      </div>
    </div>
  </nav>
</header>
`;
